tools/ua.bash

```diff
@@ -9,9 +9,9 @@ _ua_complete()
     prev_word="${COMP_WORDS[COMP_CWORD-1]}"
 
     if [ "$VERSION_ID" = "16.04" ] || [ "$VERSION_ID" == "18.04" ]; then
-        services="cc-eal cis esm-apps esm-infra fips fips-updates livepatch realtime-kernel ros ros-updates"
+        services="cc-eal cis esm-apps esm-infra fips fips-updates landscape livepatch realtime-kernel ros ros-updates"
```

Comment:
Hint that we forgot anbox here in the previous PR didn't we
Good that you remembered this

Reply:
Added in #2669